release-notes.hbs.md

```diff
@@ -29,6 +29,15 @@ This release includes the following changes, listed by component and area.
 
 - Feature description.
 
+#### <a id='1-7-0-cnrs'></a> v1.7.0 features: Cloud Native Runtimes
+
+- **New config option `contour.default_tls_secret`**: We are deprecating the current `default_tls_secret` config option.
+  This new option has the same meaning as `default_tls_secret`. Both config options are going to be supported for some releases.
```

Comment:
regarding: `Both config options are going to be supported for some releases.`

Do you think we could get a exact number? Does the deprecation policy say anything specific?

I'm wondering if we could confidently say "Both will be supported for at least 2 more releases" or something.